tools/intg-release-migrator/tools/importUtils.ts

```diff
@@ -118,15 +125,29 @@ export async function replaceImports(baseDir: string): Promise<void> {
 
   await Promise.all(
     Object.entries(pathsToPatchImports).map(async ([pathToFile, count]) => {
-      const scriptData = await fs.readFile(pathToFile, 'utf8');
+      const newPathToFile =
+        pathToFile.endsWith('.js') && !pathToFile.includes('/cjs/') && mainIsCJs
+          ? pathToFile.replace(/\.js$/, '.mjs')
```

Comment:
looks like here should be `cjs` instead of `mjs`... no?

Reply:
No, let's discuss in person